Framework/include/QualityControl/ServiceDiscovery.h

```diff
@@ -15,20 +15,27 @@ namespace o2::quality_control::core
 {
 
 /// \brief Information service for QC
+///
+/// Register a endpoint to Consul which then performs health checks it
+/// Allow to publish list of online objects
 class ServiceDiscovery
 {
   public:
-    /// Sets up CURL
+    /// Sets up CURL and health check
+    /// \param url 		Consul URL
+    /// \param id 		Unique instance ID
+    /// \param healthEndpoint	Local endpoint that is then used for health checks
+    ///				(default value it set to  <hostname>:7777)
     ServiceDiscovery(const std::string& url, const std::string& id, const std::string& healthEndpoint);
```

Comment:
you say that it has a default value, but does it ? do you mean that I can pass an empty string ?

Reply:
just to clarify, if I don't pass the third argument and try to make_unique a ServiceDiscovery there is an error.

Reply:
See here